src/mirror-client-finder/directory-finder.ts

```diff
@@ -20,6 +21,11 @@ export async function findClientDir(client: InstallableClient, desiredVersionRan
     );
     const clientVersionedDir = clientBaseDir + clientMatchedVersion;
 
+    if (client === Inputs.CRC && getOS() === "macos" && clientMatchedVersion > "1.27.0") {
+        throw new Error(`❌ ${Inputs.CRC} ${clientMatchedVersion} cannot be installed on MacOS. `
```

Comment:
macos is styled as "macOS". https://en.wikipedia.org/wiki/MacOS

Reply:
not fixed

Reply:
yeah